rest_api.py

```diff
@@ -53,11 +52,11 @@ def predict():
     df['color'] = df['color'].map(map_)
     df[logs] = df[logs].applymap(np.log)
     df[sqrts] = df[sqrts].applymap(np.sqrt)
-    X = df.values.reshape(1, -1)
+    X = df.values.reshape(-1, len(columns))
     X_scaled = scaler.transform(X)
 
-    prediction = np.round(model.predict(X_scaled)).astype(np.int8)[0]
-    return jsonify({"prediction": '{}'.format(prediction)})
+    prediction = np.round(model.predict(X_scaled)).astype(np.int8)
+    return jsonify('{}'.format(prediction))

```

Comment:
Try to explain the necessity of the '{}'.format(prediction)

Reply:
The string is converted to json , the array is not.